RELEASING.md

```diff
@@ -2,30 +2,50 @@
 
 ## Preparing for the Release
```

Comment:
Are all of these steps necessary if I wanted to do a release, or are these steps only necessary for setting up travis once?